packages/tour-kit/src/components/tour-kit-step.tsx

```diff
@@ -1,6 +1,7 @@
 /**
  * External Dependencies
  */
+import { useMobileBreakpoint } from '@automattic/viewport-react';
```

Comment:
Since this is also being used in `tour-kit-frame.tsx` would it be wise to just pass that in as a prop instead of importing it? Just a thought, not 100% sure

Reply:
Using hooks is almost always preferred over props. They cut down the API surface area of the component and flatten the props tree. Hook are like acceptable global variables, better to use them directly than pass their values down.